packages/api-server/src/cliHandlers.ts

```diff
@@ -43,60 +43,55 @@ export const apiCliOptions = {
 export const apiServerHandler = async (options: ApiServerArgs) => {
   const { port, socket, apiRootPath } = options
   const tsApiServer = Date.now()
-  process.stdout.write(c.dim(c.italic('Starting API Server...\n')))
+  process.stdout.write(c.dim.italic('Starting API Server...\n'))
 
   let fastify = createFastifyInstance()
 
   // Import Server Functions.
   fastify = await withFunctions(fastify, options)
 
-  const http = startFastifyServer({
-    port,
-    socket,
-    fastify,
-  }).ready(() => {
-    console.log(c.italic(c.dim('Took ' + (Date.now() - tsApiServer) + ' ms')))
-
-    const on = socket
-      ? socket
-      : c.magenta(`http://localhost:${port}${apiRootPath}`)
-    console.log(`API listening on ${on}`)
-    const graphqlEnd = c.magenta(`${apiRootPath}graphql`)
-    console.log(`GraphQL endpoint at ${graphqlEnd}`)
+  fastify = await startFastifyServer({ port, socket, fastify })
+
+  fastify.ready(() => {
+    console.log(c.dim.italic('Took ' + (Date.now() - tsApiServer) + ' ms'))
+
+    const apiServer = c.magenta(`${fastify.listeningOrigin}${apiRootPath}`)
+    const graphqlEndpoint = c.magenta(`${apiServer}graphql`)
+
+    console.log(`API server listening at ${apiServer}`)
+    console.log(`GraphQL endpoint at ${graphqlEndpoint}`)
+
     sendProcessReady()
   })
   process.on('exit', () => {
-    http?.close()
+    fastify?.close()
   })
 }
 
 export const bothServerHandler = async (options: BothServerArgs) => {
   const { port, socket } = options
   const tsServer = Date.now()
-  process.stdout.write(c.dim(c.italic('Starting API and Web Servers...\n')))
+  process.stdout.write(c.dim.italic('Starting API and Web Servers...\n'))
```

Comment:
Not sure why this one writes to `process.stdout` while the others use `console log`, was just trying to make minimal changes